res/Sectors/M1105/Porlock.txt

```diff
@@ -0,0 +1,446 @@
+1538                      E479000-0 Ba                           { -3 } (700-3) [0000] - -  - 033 10 NaXX M4 V           
+1631                      C555852-7 Ph Pa                        { -1 } (878-1) [6733] - -  - 824 14 NaXX M5 V           
+1632                      E787306-3 Ga Lo                        { -3 } (621+2) [1141] - -  - 703 11 NaXX F5 V           
+1635 Anpaan               A755975-B Ga Hi (Anpaan) Cx            {  3 } (A8B+1) [EC7B] - -  - 500  7 Ar   G6 V           
+1636                      D373776-6 Pi AnpaW                     { -2 } (864-4) [9523] - -  - 110  9 Ar   M1 V           
+1639                      C675402-9 Ni Pa                        { -1 } (A30-3) [736B] - N  - 523 13 NaXX M5 V           
+1640                      B250305-A De Lo Po                     {  1 } (721+1) [345A] - N  - 604 12 NaXX M8 V M1 V      
+1735                      E698000-0 Ba                           { -3 } (600+1) [0000] - -  - 020 11 Ar   G0 V           
+1736                      B457652-A Ag Ni AnpaW                  {  2 } (B56-5) [182B] - N  - 511 11 Ar   M0 V M1 V      
+1737                      E9C5165-7 Fl Lo AnpaW Re O:1635        { -3 } (501+3) [1173] - -  - 520  6 Ar   G4 V M4 V      
+1831                      E512425-7 Ic Ni                        { -3 } (931+0) [4128] - -  - 120  9 NaXX M1 V           
+1833 Speiskex             C758753-9 Ag AnpaW                     {  1 } (866+3) [5898] - N  - 623 12 Ar   G3 V           
+1835                      C643526-9 Ni Po AnpaW                  { -1 } (F41-2) [143A] - N  - 805 11 Ar   M1 V           
+1839                      X368203-2 Lo                           { -3 } (A11+0) [6153] - -  - 701 11 NaXX K1 V           
+1932                      E400000-0 Ba Va                        { -3 } (900-3) [0000] - -  - 003 10 NaXX M3 V           
+1935                      D788723-5 Ag Ri                        {  0 } (867-1) [9749] - N  - 710  5 Ar   K4 V           
+1936                      E75A311-9 Lo Wa                        { -2 } (721-2) [7176] - -  - 804 15 Ar   G3 V M5 V      
+1939                      C64A324-9 Lo Wa                        { -1 } (921+1) [5217] - -  - 514 11 NaXX G7 V           
+2032                      E493502-5 Ni                           { -3 } (540+3) [3298] - -  - 805 10 NaXX M7 V           
+2034                      C231741-9 Na Po AnpaW                  {  0 } (763+0) [7787] - -  - 902 14 Ar   F8 V           
+2038                      E577461-5 Ni Pa O:2040                 { -3 } (930-2) [4186] - N  - 303  9 Hd   M2 V M3 V      
+2040                      B88A853-9 Ri Ph Wa                     {  2 } (B77+4) [7A1C] - N  - 814 17 NaXX G1 V           
+2134                      C522628-9 He Na Ni Po AnpaW            { -1 } (455-1) [1557] - N  - 900  8 Ar   M5 V           
+2138               C455786-9 Ag                           {  1 } (B6A-3) [482A] - -  - 614 16 Hd   M3 V           
```

Comment:
This line is misaligned. Not really a problem normally, but my parser chokes on it.